source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfig.mm

```diff
@@ -341,4 +335,61 @@ + (NSArray *)getConstraintsForImageAlignment:(HorizontalAlignment)alignment
     return dateParsedString;
 }
 
++ (UIColor *)convertHexColorCodeToUIColor:(const std::string&)hexColorCode
+{
+    if((hexColorCode.length() < 2) || (hexColorCode.at(0) != '#') || !isxdigit(hexColorCode.at(1)) ||
```

Comment:
>isxdigit(hexColorCode.at(1) [](start = 70, length = 27)

Should this validate that all digits are hex?

Reply:
thanks!


---
In reply to: [220330945](https://github.com/Microsoft/AdaptiveCards/pull/1954#discussion_r220330945) [](ancestors = 220330945)